docs/topics/request-response.rst

```diff
@@ -24,7 +24,7 @@ below in :ref:`topics-request-response-ref-request-subclasses` and
 Request objects
 ===============
 
-.. class:: Request(url[, callback, method='GET', headers, body, cookies, meta, encoding='utf-8', priority=0, dont_filter=False, errback])
+.. class:: Request(url[, callback, method='GET', headers, body, cookies, meta, encoding='utf-8', priority=0, dont_filter=False, errback, flags = null])
```

Comment:
yes thank you

On Wed, Sep 7, 2016 at 11:19 AM, Paul Tremberth <notifications@github.com>
wrote:

> In docs/topics/request-response.rst
> <https://github.com/scrapy/scrapy/pull/2211#discussion_r77778101>:
>
> > @@ -24,7 +24,7 @@ below in :ref:`topics-request-response-ref-request-subclasses` and
> >  Request objects
> >  ===============
> >
> > -.. class:: Request(url[, callback, method='GET', headers, body, cookies, meta, encoding='utf-8', priority=0, dont_filter=False, errback])
> > +.. class:: Request(url[, callback, method='GET', headers, body, cookies, meta, encoding='utf-8', priority=0, dont_filter=False, errback, flags = null])
>
> @foromer4 <https://github.com/foromer4> , do I understand that this PR
> can be closed?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/scrapy/scrapy/pull/2211/files/3eb09f7483965bc975e82693acc54507b5aa7347#r77778101>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AKA76FsMo1yez5Wtx2l8UGdXwYDCO5Hkks5qnnOJgaJpZM4JyZgj>
> .
>